ncm-accounts/src/main/pan/components/accounts/functions.pan

```diff
@@ -42,6 +42,11 @@ function is_user_or_group = {
         );
     };
     pref = "/software/components/accounts";
+    # not all sites use ncm-accounts, e.g. users and groups may be provisioned
```

Comment:
if this is the case, this function also needs to be moved out this component (imho). how would this function be available otherwise.

Reply:
functions.pan is standalone so I have no problem including it even without accounts/config.